api/db/migrations/20250807115444_new-certif-flash-config-table.js

```diff
@@ -0,0 +1,64 @@
+const TABLE_NAME = 'certification-flash-algorithms';
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+const up = async function (knex) {
+  await knex.schema.createTable(TABLE_NAME, function (table) {
+    const comment =
+      'This table holds the Certification Flash algorithm configurations ' +
+      'It drives the certification assessments challenge selections and ' +
+      'how a certification assessment s challenges are translated into a scoring';
+    table.comment(comment);
+
+    table.increments('id').primary().comment('Technical identifier');
+
+    table
+      .dateTime('startingDate')
+      .notNullable()
+      .defaultTo(knex.fn.now())
+      .comment('Inclusive. This algorithm version is applicable starting this date.');
+
+    table.dateTime('expirationDate').comment('Exclusive. This algorithm version is applicable until this date.');
+
+    table.integer('maximumAssessmentLength').comment('Maximum number of challenges for an assessment');
+    table
+      .integer('challengesBetweenSameCompetence')
+      .comment('Number of challenges before presenting again a challenge in the same Pix competences');
+
+    table.boolean('limitToOneQuestionPerTube').comment('Whether or not we allow multiple challenges for one Pix tube');
+
+    table
+      .boolean('enablePassageByAllCompetences')
+      .comment('Force to present at least one challenge from every Pix competence');
+
+    table
+      .integer('variationPercent')
+      .comment('The allowed peak-to-peak amplitude in the variation of the difficulty between two challenges');
+
+    table
+      .integer('configuration')
+      .comment('The allowed peak-to-peak amplitude in the variation of the difficulty between two challenges');
+
+    table
+      .jsonb('scoring-configuration')
+      .nullable()
+      .comment("Defines the mesh levels and boundaries of the assessment's global scoring");
+
+    table
+      .jsonb('competences-configuration')
```

Comment:
```suggestion
      .jsonb('competences-scoring-configuration')
```
Pour garder la notion de scoring dans l'intitulé de la colonne

Reply:
Euh du coup je mets `global-scoring-configuration` pour l'autre colonnes ?

Reply:
Vu ensemble, je vais mettre global-scoring-configuration

Reply:
Done